Lib/test/.ruff.toml

```diff
@@ -4,11 +4,11 @@ select = [
 ]
 extend-exclude = [
     # Failed to lint
-    "badsyntax_pep3120.py",
+    "tokenizedata/badsyntax_pep3120.py",
```

Comment:
Could you move this below `encoded_modules/module_koi8_r.py`, so that the items in the "Failed to lint" section are in alphabetical order?